dotnet/src/InternalUtilities/test/HttpMessageHandlerStub.cs

```diff
@@ -24,12 +25,15 @@ internal sealed class HttpMessageHandlerStub : DelegatingHandler
 
     public HttpResponseMessage ResponseToReturn { get; set; }
 
+    public Queue<HttpResponseMessage> ResponseQueue { get; } = new();
+
     public HttpMessageHandlerStub()
     {
-        this.ResponseToReturn = new HttpResponseMessage(System.Net.HttpStatusCode.OK)
-        {
-            Content = new StringContent("{}", Encoding.UTF8, MediaTypeNames.Application.Json)
-        };
+        this.ResponseToReturn =
+            new HttpResponseMessage(System.Net.HttpStatusCode.OK)
+            {
+                Content = new StringContent("{}", Encoding.UTF8, MediaTypeNames.Application.Json),
+            };
```

Comment:
Not sure if there is a need to change this formatting. My preference is current version.

Reply:
Updated